README.rst

```diff
@@ -0,0 +1,89 @@
+===============================================
+``gordon``: Event-driven Cloud DNS registration
+===============================================
+
+.. desc-begin
+
+Service to consume hostname change events and update records for a 3rd party DNS provider.
```

Comment:
Can "hostname change events" be substituted for "compute operation events"?  And also "update" to "create or update"?

Reply:
I'm fine with changing how it reads, but it's not necessarily compute operation events that gordon will use. It might be events from some other pubsub system. Thoughts?